whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
 %-
 % ====================================================================
 
+\newcommand{\red}[1]{\textcolor{red}{#1}}
+
 \section{Weak Lensing}
 \def\secname{wl}\label{sec:\secname}
 
-\credit{StephenRidgway},
 \credit{jmeyers314},
-\credit{tonytyson}.
+\credit{tonytyson},
+\credit{StephenRidgway}.
 
-Much of LSST cosmology may be limited by systematic errors rather than
-photon signal-to-noise. This is especially true of weak gravitational
-lensing,  which relies on very accurate (\ie low bias), but very low
-signal-to-noise, measurements of the shapes of galaxies, and high
-signal-to-noise measurements of PSF calibration stars. As outlined in the SRD,
-uniformity of seeing in the bands used for WL and special observing strategies 
-are required in order to reduce additive and multiplicative shear systematics.
+Much of LSST cosmology may be limited by systematic rather than statistical
+errors.  This is especially true of weak gravitational lensing, which relies on
+very accurate (\ie low bias), but very noisy, measurements of the shapes of many
```

Comment:
We need very accurate ensemble shear estimates, not very accurate shape measurements of individual galaxies.